_data/current.yml

```diff
@@ -22,6 +22,13 @@
   twitter: the_test_tribe
   status: Registration is Open and Free
 
+  name: Test Coast 2024
```

Comment:
Wouldn't this require a `-` before the `name`?

Reply:
Yes you are right